### PR TITLE
user see profile

### PR DESCRIPTION
**Pull request by TuanAnhVH**
user see profile
Screenshot from 2020-11-24 11-30-49

### CODE REVIEW

**Review by OE38-LeHoangHa — APPROVED** [review]

**Comment by TuanAnhVH** [comment]
ready

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+class UsersController < ApplicationController
+  before_action :load_user
+  before_action :correct_user
```
Comment (vanvtt-0952) [inline app/controllers/users_controller.rb]: Sao ko gộp 2 line này thành 1 line chú

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+          <td><%= t "base.address" %>:</label></td>
+          <td><%= @user.address %></td>
+        </tr>
+        <tr>
+          <td><%= t "base.phone" %>:</label></td>
+          <td><%= @user.phone %></td>
+        </tr>
+        <tr>
+          <td><%= t ".created_at" %>:</label></td>
+          <td><%= @user.created_at %></td>
```
Comment (vanvtt-0952) [inline app/views/users/show.html.erb]: format datetime đi chú

**Comment by TuanAnhVH** [comment]
ready

**Pull request merged**